src/DurableTask.Core/Common/Utils.cs

```diff
@@ -80,6 +81,18 @@ public static string Truncate(this string input, int maxLength)
             return input;
         }
 
+        internal static JArray ConvertToJArray(string input)
+        {
+            JArray jArray = new JArray();
```

Comment:
I suggest removing the initial allocation since we're going to throw it away down below.
```suggestion
            JArray jArray;
```